app/controllers/api/v1/sol_prices_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SolPricesController < BaseController
+      def index
+        from = (index_params[:from] || 30.days.ago).to_datetime
+        to = (index_params[:to] || from + 30.days).to_datetime
```

Comment:
`from + 30.days` 
Why not just `Time.now`?

Reply:
I wanted it to work well even if someone uses only `from` param